### DISCUSSION TITLE
fa/manager/installation-and-setup/Installation-prerequisites/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4001] [Content Help کمک به اصلاح محتوا]
# fa/manager/installation-and-setup/Installation-prerequisites/

The proxy manager platform.

https://hiddify.com/fa/manager/installation-and-setup/Installation-prerequisites/

**Comment by Mamon229** [comment]
Отлично всё сделал как положено

**Reply by on-Ruto** [reply]
russian?

**Comment by uktam7577887-blip** [comment]
Salo

**Comment by uktam7577887-blip** [comment]
Salom
